src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -23,4 +29,38 @@ private Et3ResponseHelper() {
     public static String formatClaimantNameForHtml(CaseData caseData) {
         return String.format(CLAIMANT_NAME_TABLE, caseData.getClaimant());
     }
+
+    /**
+     * Validates that the employment start date is in the past and not after 
+     * the employment end date if both dates are provided.
+     * @param caseData data for the current case
+     * @return List of validation errors encountered
+     */
+    public static List<String> validateEmploymentDates(CaseData caseData) {
+        ArrayList<String> errors = new ArrayList<>();
+        String startDateStr = caseData.getEt3ResponseEmploymentStartDate();
+        String endDateStr = caseData.getEt3ResponseEmploymentEndDate();
+
+        if (isNullOrEmpty(startDateStr)) {
```

Comment:
Do we need an error message if start date is empty? Same with end date?

Reply:
Start Date and End date are optional and validation should only take place once values are inputted

Reply:
This is stated in the JavaDoc too